Assets/Standard Assets/Characters/FirstPersonCharacter/Scripts/RigidbodyFirstPersonController.cs

```diff
@@ -177,7 +177,7 @@ private void FixedUpdate()
             }
             else
             {
-                m_RigidBody.drag = 0f;
+                m_RigidBody.drag = 0.5f;
```

Comment:
I'm interested to know what this does considering the difficulty I've had with the standard FPS controller!